.eslintrc.json

```diff
@@ -1,34 +1,44 @@
 {
-    "extends": "airbnb",
-		"indent": ["error", "tab"],
-		"env": {
-        "es6": true,
-        "node": true,
-        "browser": true
-    },
-    "parserOptions": {
-        "ecmaFeatures": {
-            "experimentalObjectRestSpread": true,
-            "jsx": true
-        },
-        "sourceType": "module"
-    },
-    "plugins": [
-        "react",
-        "jsx-a11y",
-        "import"
-    ],
-    "rules": {
-      "react/sort-comp": [1, {
-          "order": [
-            "constructor",
-            "lifecycle",
-            "everything-else",
-            "render"
-          ]
-        }
-      ],
-      "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
-      "import/no-dynamic-require": 0
-    }
-}
\ No newline at end of file
+	"parser": "babel-eslint",
```

Comment:
True that. It is used for flow-types, which I use on my clone but strip them before pushing to github.